.github/workflows/node-ci.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   test-build-release:
     env:
-      IS_PUSH_TO_MASTER: ${{ github.event.push && github.ref == 'refs/heads/master' }}
+      IS_PUSH_TO_MASTER: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
```

Comment:
Why not just break out the storybook action into a new yml file and have it only run on pushes to master?

Reply:
I will keep your suggestion in mind for subsequent work that might be on a different repo.
The things to keep in mind are
- the repeating the checkout and install process
- ensuring that the build steps have succeeded
- or passing outputs around.

Reply:
Also see [this post](https://github.community/t/how-do-i-specify-job-dependency-running-in-another-workflow/16482)